src/overlays/actors/ovl_En_Am/z_en_am.c

```diff
@@ -127,48 +127,48 @@ static ColliderQuadInit sQuadInit = {
     { { { 0.0f, 0.0f, 0.0f }, { 0.0f, 0.0f, 0.0f }, { 0.0f, 0.0f, 0.0f }, { 0.0f, 0.0f, 0.0f } } },
 };
 
-typedef enum ArmosDamageEffect {
-    /*  0 */ AM_DMGEFF_NONE, // used by anything that can't kill the armos
-    /*  1 */ AM_DMGEFF_NUT,
-    /*  6 */ AM_DMGEFF_STUN = 6, // doesn't include deku nuts
-    /* 13 */ AM_DMGEFF_ICE = 13,
-    /* 14 */ AM_DMGEFF_MAGIC_FIRE_LIGHT,
-    /* 15 */ AM_DMGEFF_KILL // any damage source that can kill the armos (and isn't a special case)
-} ArmosDamageEffect;
+typedef enum ArmosDamageReaction {
+    /*  0 */ AM_DMG_REAC_NONE, // used by anything that can't kill the armos
```

Comment:
```suggestion
    /*  0 */ AM_DMG_REACT_NONE, // used by anything that can't kill the armos
```

In English, REAC is an unusual way to abbreviate the word reaction.

Reply:
atp could also write REACTION in full, it adds two line breaks after formatting https://github.com/zeldaret/oot/compare/main...Dragorn421:oot:doc_rename_damageReaction_REACTION